dev-workflow-ui/webContent/includes/template/notifications.xhtml

```diff
@@ -0,0 +1,63 @@
+<ui:composition xmlns="http://www.w3.org/1999/xhtml" xmlns:f="http://xmlns.jcp.org/jsf/core"
+  xmlns:h="http://xmlns.jcp.org/jsf/html" xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+  xmlns:c="http://xmlns.jcp.org/jsp/jstl/core" xmlns:ic="http://ivyteam.ch/jsf/component"
+  xmlns:p="http://primefaces.org/ui" xmlns:pe="http://primefaces.org/ui/extensions">
+
+<p:commandLink id="showNotifications">
+  <i class="topbar-icon pi pi-bell"/>
+  <div class="notification-badge">
+    <h:outputText id="unreadNotifications" value="#{notificationBean.unreadNotifications}" />
+  </div>
+</p:commandLink>
```

Comment:
were you not able to use primefaces badges?

https://www.primefaces.org/showcase/ui/misc/badge.xhtml?jfwid=4c15e